openedx/features/edly/tests/factories.py

```diff
@@ -0,0 +1,50 @@
+"""
+Model factories for unit testing views or models.
+"""
+
+from django.contrib.sites.models import Site
+from factory import SelfAttribute, Sequence, SubFactory
+from factory.django import DjangoModelFactory
+from organizations.tests.factories import OrganizationFactory
+
+from openedx.features.edly.models import EdlyOrganization, EdlySubOrganization
+
+
+class EdlyOrganizationFactory(DjangoModelFactory):
+    """
+    Factory class for Organization model
```

Comment:
First line is a summary in docstring and it should end with a dot `.`
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings

Reply:
Done